platform/ios/src/MGLCalloutView.h

```diff
@@ -46,6 +46,22 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)dismissCalloutAnimated:(BOOL)animated;
 
+@optional
+
+/**
+ A Boolean value indicating whether the callout view should be anchored to
+ the corresponding annotation. The exact position can be adjusted by 
+ overriding -[UIView setCenter:]
```

Comment:
Can you point out the default behavior in the documentation for both properties (so developers know whether they need to implement them or not)?